wrapper.js

```diff
@@ -177,6 +177,25 @@ exports.generateKeys = function generateKeys(bitcoinNetwork) {
     return JSON.parse(lib.rgblib_generate_keys(bitcoinNetwork));
 };
 
+exports.restoreBackup = function (backupPath, password, targetDir) {
+    const params = {
+        assetId,
+        amount,
+        durationSeconds,
+        transportEndpoints,
+        minConfirmations,
+    };
+    const expectedTypes = {
+        assetId: "string?",
+        amount: "u64?",
+        durationSeconds: "u32?",
+        transportEndpoints: "array[string]",
+        minConfirmations: "u8",
+    };
+    validateTypes(params, expectedTypes);
```

Comment:
I think you forgot to change the parameters here, we should validate the ones of the `restoreBackup` method

Reply:
🫠